ublox/src/ubx_packets/cfg_val.rs

```diff
@@ -42,6 +42,16 @@ impl KeyId {
     pub const fn item_id(&self) -> u8 {
         self.0 as u8
     }
+
+    pub fn extend_to<T>(&self, buf: &mut T) -> usize
```

Comment:
Is now a good time to start more rigorously documenting the API? 😄 

```suggestion
    /// Extend a buffer with the contents of [KeyId].
    pub fn extend_to<T>(&self, buf: &mut T) -> usize
```

Reply:
true